en/orm/associations.rst

```diff
@@ -357,6 +357,33 @@ syntax::
         }
     }
 
+Sometimes you may want to configure composite keys in your associations::
+
+    // Within ArticlesTable::initialize() call
+    $this->hasMany('Reviews', [
+        'foreignKey' => [
+            'article_id',
+            'article_hash'
+        ]
+    ]);
+
+In the example above, we have passed an array containing the desired composite
+keys to ``foreignKey``. Maybe you need to specify different
+binding fields than the defaults, so you can setup it manually in your
+``bindingKeys`` array::
+
+    // Within ArticlesTable::initialize() call
+    $this->hasMany('Reviews', [
+        'foreignKey' => [
+            'article_id',
+            'article_hash'
+        ],
+        'bindingKey' => [
+            'whatever_id',
+            'whatever_hash'
+        ]
+    ]);
```

Comment:
We should probably explain what bindingKey is an how it differs from foreignKey.